dist/common/scripts/scylla_setup

```diff
@@ -469,4 +469,8 @@ if __name__ == '__main__':
             print(' SELinux.')
 
         if dist_name() == 'Ubuntu':
-            run('apt-get install -y hugepages')
+            if ubuntu_version() == '20.04':
```

Comment:
Let's check for later release too, so we don't hit the same problem in April 2022.

Reply:
@avikivity supported later version.